docs/Documentation/UserGuideV0.7/3-Operation Manual.md

```diff
@@ -0,0 +1,797 @@
+
+```
+IoTDB > set storage group to root.ln
+IoTDB > set storage group to root.sgcc
+```
+
+We can thus create two storage groups using the above two SQL statements.
+
+It is worth noting that when the path itself or the parent/child layer of the path is already set as a storage group, the path is then not allowed to be set as a storage group. For example, it is not feasible to set `root.ln.wf01` as a storage group when there exist two storage groups `root.ln` and `root.sgcc`. The system will give the corresponding error prompt as shown below:
+
+```
+IoTDB> set storage group to root.ln.wf01
+error: The prefix of root.ln.wf01 has been set to the storage group.
+```
+
+### Show Storage Group
+After the storage group is created, we can use the [SHOW STORAGE GROUP](Chapter5,showstoragegroup) statement to view all the storage groups. The SQL statement is as follows:
+
+```
+IoTDB> show storage group
+```
+
+The result is as follows:
+<center><img style="width:80%" src="./fig/Detail-2.jpg"></center>
+
+### Timeseries Creation
+According to the storage model selected before, we can create corresponding timeseries in the two storage groups respectively. The SQL statements for creating timeseries are as follows:
+
+```
+IoTDB > create timeseries root.ln.wf01.wt01.status with datatype=BOOLEAN,encoding=PLAIN
+IoTDB > create timeseries root.ln.wf01.wt01.temperature with datatype=FLOAT,encoding=RLE
+IoTDB > create timeseries root.ln.wf02.wt02.hardware with datatype=TEXT,encoding=PLAIN
+IoTDB > create timeseries root.ln.wf02.wt02.status with datatype=BOOLEAN,encoding=PLAIN
+IoTDB > create timeseries root.sgcc.wf03.wt01.status with datatype=BOOLEAN,encoding=PLAIN
+IoTDB > create timeseries root.sgcc.wf03.wt01.temperature with datatype=FLOAT,encoding=RLE
+```
+
+It is worth noting that when in the CRATE TIMESERIES statement the encoding method conflicts with the data type, the system will give the corresponding error prompt as shown below:
+
+```
+IoTDB> create timeseries root.ln.wf02.wt02.status WITH DATATYPE=BOOLEAN, ENCODING=TS_2DIFF
+error: encoding TS_2DIFF does not support BOOLEAN
+```
+
+Please refer to [Encoding](Chapter2,encoding) for correspondence between data type and encoding.
+
+### Show Timeseries
+
+Currently, IoTDB supports two ways of viewing timeseries:
+
+* SHOW TIMESERIES statement presents all timeseries information in JSON form 
+* SHOW TIMESERIES <`Path`> statement returns all timeseries information and the total number of timeseries under the given <`Path`>  in tabular form. timeseries information includes: timeseries path, storage group it belongs to, data type, encoding type.  <`Path`> needs to be a prefix path or a path with star or a timeseries path. SQL statements are as follows:
+
+```
+IoTDB> show timeseries root
+IoTDB> show timeseries root.ln
+```
+
+The results are shown below respectly:
+
+<center><img style="width:80%" src="./fig/Detail-3.jpg"></center>
```

Comment:
I think we should just paste the result as plain text in code block to the documentation. These screenshots will cause the documentation hard to maintain and that is also not elegant in style (font. size and background color, etc.)

Reply:
Yes, we will use plain text instead of picture in future document.